docs/tutorial/10_ether-faucet.md

```diff
@@ -0,0 +1,119 @@
+# 10. Create a Faucet
+
+During the development and testing of contracts and dapps it is useful to modify the blockchain's state to account for previous history. For example,
```

Comment:
Because I want to teach not just "this is how we create an ether faucet", but "this is how we modify the blockchain's state to be what it should be at the beginning of whatever it is we're testing".